haskell/private/haskell_impl.bzl

```diff
@@ -316,6 +316,13 @@ def _haskell_binary_common_impl(ctx, is_test):
             binary,
         ] + mix_runfiles + srcs_runfiles + java.inputs.to_list()
 
+        # This provider will trigger the coverage generation for .lcov files
+        instrumented_files_info = [coverage_common.instrumented_files_info(
+            ctx,
```

Comment:
Does this not need to set any of the [other parameters](https://docs.bazel.build/versions/main/skylark/lib/coverage_common.html#parameters) like `source_attributes`?